config/config.go

```diff
@@ -923,38 +924,38 @@ func ParseConfig(ctx *ParsingContext, file *hclparse.File, includeFromChild *Inc
 		// process.
 		retrievedOutputs, err := decodeAndRetrieveOutputs(ctx, file)
 		if err != nil {
-			return nil, err
+			errs = errs.Append(err)
 		}
 
 		ctx.DecodedDependencies = retrievedOutputs
 	}
 
 	evalContext, err := createTerragruntEvalContext(ctx, file.ConfigPath)
 	if err != nil {
-		return nil, err
+		errs = errs.Append(err)
 	}
 
 	// Decode the rest of the config, passing in this config's `include` block or the child's `include` block, whichever
 	// is appropriate
 	terragruntConfigFile, err := decodeAsTerragruntConfigFile(ctx, file, evalContext)
 	if err != nil {
-		return nil, err
+		errs = errs.Append(err)
 	}
 
 	if terragruntConfigFile == nil {
```

Comment:
Downstream usage of `terragruntConfigFile` is not checking for terragruntConfigFile nil